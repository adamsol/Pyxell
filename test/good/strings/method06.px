
s = "abc"
print s.has('c'), s.has('a'), s.has('b')
print s.has('A'), s.has('d')

print "aAbcC".has("abc")
print "abcd".has("abc")
print "XYxyZ".has("xy")
print "xyz".has("xyz")
print "xy".has("yx")
print ".".has("")
print "".has("")
