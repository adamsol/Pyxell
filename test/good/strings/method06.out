true true true
false false
false
true
true
true
false
true
true
